src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseData.java

```diff
@@ -352,7 +352,6 @@
     private final String boExaminationChecklistRequestQA;
     private final String applicationSubmittedDate;
     private final List<CollectionMember<ScannedDocument>> scannedDocuments;
-    private final String evidenceHandled;
```

Comment:
please compare this java file with master , dont see any of the changes made here are required . this file shouldnt appear in the change set for u